vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -453,6 +463,16 @@ void AdsImpl::OnTabUpdated(
     active_tab_id_ = tab_id;
     previous_tab_url_ = active_tab_url_;
     active_tab_url_ = url;
+
+    UserActivityType user_activity_type;
+
+    if (url == "chrome://newtab/") {
```

Comment:
should this be `brave://newtab`?

Reply:
also thinking out loud - this won't trigger if a user cmd + clicks on a link, which is fine I think. We could always add something like `kOpenedLinkInNewTab` if deemed useful.

Reply:
Over time we can definitely add more event types